libraries/pbrlib/genglsl/mx_dielectric_bsdf.glsl

```diff
@@ -21,7 +21,15 @@ void mx_dielectric_bsdf_reflection(vec3 L, vec3 V, vec3 P, float occlusion, floa
     vec2 safeRoughness = clamp(roughness, M_FLOAT_EPS, 1.0);
     float avgRoughness = mx_average_roughness(safeRoughness);
 
-    FresnelData fd = tf.thickness > 0.0 ? mx_init_fresnel_dielectric_airy(ior, tf.thickness, tf.ior) : mx_init_fresnel_dielectric(ior);
+    FresnelData fd;
```

Comment:
Reverted other changes which we're not needed / incorrect.
This is the only implicit cast code fix required.